components/card.js

```diff
@@ -1,5 +1,57 @@
-export default ({children}) => (
-    <div>
-        {children}
-    </div>
+import React from 'react'
+import styled from 'styled-components'
+import Button from './button'
+
+const Card = styled.div`
+    background-color: #F8F8F8;
+`
+
+const Header = styled.div`
+    background-color: #EDEDED;
+    padding: 24px 40px;
+    display: flex;
+    align-items: center;
+`
+
+const Title = styled.h1`
+    font-size: 24px;
+    margin: 0;
+`
+
+const Subtitle = styled.p`
+    font-size: 16px;
+    margin: 0;
+    margin-left: 24px;
+`
+
+const ButtonContainer = styled.div`
+    margin-left: auto;
+`
+
+const Content = styled.div`
+    padding: 40px;
+`
+
+export default ({title, subtitle, author, buttonText, onClick, ctaType, children}) => (
+    <Card>
+        <Header>
+            <Title>
+                {title}
+            </Title>
+            {
+                subtitle &&
+                <Subtitle>
+                    {subtitle} by <b>{author}</b>
```

Comment:
are we going to include when the person edited?

Reply:
I was thinking that would be up to the consumer to pass the time